content/_index.md

```diff
@@ -1,58 +1,62 @@
 ---
 hero:
   title: The Free Software Media System
-  text: Jellyfin puts you in control of managing and streaming your media. There are no strings attached, no premium licenses or features, and no hidden agendas.
+  text: Jellyfin is the volunteer-built media solution that puts *you* in control of your media. Stream to any device from your own server, with no strings attached. Your media, your server, your way.
```

Comment:
Maybe `community-built` instead of `volunteer-built`?
Also `Your media, your way` is Emby slogan, why do we want to copy that now?